src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Modal from '../Modal/Modal';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+
+export const ModalCancelEvent = ({ cancelMessage }) => {
+  return (
+    <div>
+      <Modal title="Cancel Event">
+        <p>Are you sure you want to cancel</p>
+        <Button
+          className="modal-cancel-event-button"
+          appearance="danger"
+          onClick={cancelMessage}
```

Comment:
It is common to prefix callbacks with `on` -> so this becomes `onCancelMessage`.
Otherwise, if you look at the name it might seem as if it is a text you pass to the component.

Reply:
renamed